config/sidekiq.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
It will be used when sidekiq is active, do you mean adding a note stating it does not matter for delayed_job?